org.sonarlint.eclipse.ui/plugin.xml

```diff
@@ -579,15 +579,6 @@
                  icon="icons/full/marker_resolution16/disablerule.png"
                  style="push"
                  mnemonic="u">
-          <visibleWhen>
```

Comment:
Why was it removed here? I'm currently not able to see the reason behind that change.

Reply:
Because I moved it under the handler declaration (2nd addition in the PR)